src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,38 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * The card is unique to each customer.
+ */
+public final class LoyaltyCard {
+    /**
+     * Number is the card's id.
+     */
+    private final String number;
+
+    /**
+     * Creating a new card with the passed number.
```

Comment:
“Creates...”. Please read how other people write Javadocs, e.g. Guava. It should give you idea on how people do it.